src/main/java/edu/greenblitz/tobyDetermined/subsystems/LED.java

```diff
@@ -31,45 +30,51 @@ public static void init(){
         }
     }
 
-    public void setColor(Color color) {
-        for (int i = 0; i < this.ledBuffer.getLength(); i++) {
-            this.ledBuffer.setLED(i, color);
-            SmartDashboard.putNumber("led num", i);
-        }
-        this.addressableLED.setData(ledBuffer);
-    }
     public void setColor(int i, Color color) {
         this.ledBuffer.setLED(i, color);
-        this.addressableLED.setData(ledBuffer);
     }
 
+    public void setColor (int startIndex,int endIndex, Color color){
+        for (int i = startIndex; i < endIndex; i++) {
+            setColor(i,color);
+        }
+    }
+
+    public void setColor (RobotMap.LED.Sections section, Color color){
```

Comment:
I think, and this is in general for all of the setColor functions, that the Color should be the first param in all of the functions, this is the main argument for the function, and it is not clear why a number or section is the first thing to appear on a setColor function.

I would also mention that I agree with Nitzan, that a good overload would be to have just a color, and expect it be on the entire range.